### PR TITLE
EIP-7594: add custody settings config

### PR DESCRIPTION
**Pull request by Savid**

### CODE REVIEW

File: configs/mainnet.yaml [hunk 1]
```diff
@@ -159,6 +159,8 @@ NUMBER_OF_COLUMNS: 128
 MAX_CELLS_IN_EXTENDED_MATRIX: 768
 DATA_COLUMN_SIDECAR_SUBNET_COUNT: 32
 MAX_REQUEST_DATA_COLUMN_SIDECARS: 16384
+SAMPLES_PER_SLOT: 8
+CUSTODY_REQUIREMENT: 1
```
Comment (hwwhww) [inline configs/mainnet.yaml:163]:
```suggestion
CUSTODY_REQUIREMENT: 1
TARGET_NUMBER_OF_PEERS: 70
```

File: configs/minimal.yaml [hunk 2]
```diff
@@ -158,6 +158,8 @@ NUMBER_OF_COLUMNS: 128
 MAX_CELLS_IN_EXTENDED_MATRIX: 768
 DATA_COLUMN_SIDECAR_SUBNET_COUNT: 32
 MAX_REQUEST_DATA_COLUMN_SIDECARS: 16384
+SAMPLES_PER_SLOT: 8
+CUSTODY_REQUIREMENT: 1
```
Comment (hwwhww) [inline configs/minimal.yaml:162]:
```suggestion
CUSTODY_REQUIREMENT: 1
TARGET_NUMBER_OF_PEERS: 70
```

File: configs/minimal.yaml [hunk 3]
```diff
@@ -158,6 +158,9 @@ NUMBER_OF_COLUMNS: 128
...
+TARGET_NUMBER_OF_PEERS: 70
```
Comment (AgeManning) [inline configs/minimal.yaml:163]: Do we need the target number of peers in the config? 

Currently I think most clients expose this via a CLI flag. Each client sets their own defaults based on node-resources and misc tradeoffs with performance. I think we probably still want to leave this up to client implementations to set

**Review by dapplion — COMMENTED** [review]
Re including this values in the spec:
- `SAMPLES_PER_SLOT`: As of today sampling is only used for a node to influence its own view on what's safe. This is subjective and different users may have different safety requirements. I would exclude this value from the config
- `CUSTODY_REQUIREMENT`: Only makes sense to add to the spec if this is the minimum custody per NodeID, enforceable with peer scoring.
- `TARGET_NUMBER_OF_PEERS`: Clients allow to customize peer counts. Clients may choose to show warnings if the user selects a value less than this. Then I'm not sure of the usefulness of having this parameter in the spec. Unless we expect different networks to have widely different peers needs and it being convenient to standardized those warning messages

**Review by hwwhww — APPROVED** [review]

**Pull request merged**